contributing/community/review-comments.rst

```diff
@@ -0,0 +1,177 @@
+Respectful Review Comments
+==========================
+
+:doc:`Reviewing issues and pull requests </contributing/community/reviews>`
+is a great way to get started with contributing to the Symfony community.
+Anyone can do it! But before you give a comment, take a step back and think,
+is what you are about to say actually what you intend?
+
+Communicating over the Internet with nothing but text can pose a
+big challenge, especially if you remember that the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+ideas and opinions.
+
+Not everyone speaks English or is able to use a keyboard. Some might
+have Dyslexia or similar conditions that affect their writing.
```

Comment:
should `dyslexia` actually have a capital letter when being in the middle of a sentence ?

Reply:
No, it should be lower case.